src/elements/content-preview/ContentPreview.js

```diff
@@ -538,7 +544,7 @@ class ContentPreview extends PureComponent<Props, State> {
             };
         }
 
-        onMetric(metrics);
+        this.props.logger.onPreviewMetric(metrics);
```

Comment:
We should be moving to de-structuring (even for single), thats the eslint rule we need to enable in the near future. Currently its turned off but we shouldn't add more.

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/destructuring-assignment.md